### PR TITLE
Search

### PR DESCRIPTION
**Pull request by henriklovhaug**

### CODE REVIEW

**Comment by henriklovhaug** [comment]
@Areskiko

**Comment by henriklovhaug** [comment]
Hook test

**Comment by henriklovhaug** [comment]
Hook test 2

**Comment by henriklovhaug** [comment]
Test 3

File: rust-stonk/tests/test_stockfinder.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+extern crate rust_stonk;
```
Comment (ghost) [inline rust-stonk/tests/test_stockfinder.rs]: Unit tests should be in the same file as the definition being tested

Comment (ghost) [inline rust-stonk/tests/test_stockfinder.rs]: Integration tests have their own rule. Please see: https://doc.rust-lang.org/book/ch11-03-test-organization.html

**Review by mack-stefan — CHANGES_REQUESTED** [review]

File: rust-stonk/tests/test_stockfinder.rs (hunk 1 above)
Comment (henriklovhaug) [reply rust-stonk/tests/test_stockfinder.rs]: Will not add at current time

**Review by mack-stefan — APPROVED** [review]

**Pull request merged**